### DISCUSSION TITLE
Custom `list-disc` color

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [tailwindlabs/tailwindcss#9081] [Help] [answered]
I would like to customize the color of the bullets in my list. Here is my code:

```
<div className="p-5">
        <ul className="prose list-disc pb-4">
               <li></li>
                  ...
        </ul>
</div>
```

How would I go about that?

I tried this:

`global.css`

```
@layer base {
  list-disc {
    @apply text-gray-900;
  }
}
```

To no avail... What am I missing?

**Comment by wongjn** [comment, marked as answer]
In CSS, we can use the [`::marker` pseudo element](https://developer.mozilla.org/en-US/docs/Web/CSS/::marker) to customize native list bullets to a certain extent. Fortunately for us, in Tailwind, we can use the `marker:` variant to target the bullets. To change the color of the bullets, we change the `color` property of the `::marker` pseudo element:

```jsx
<div className="p-5">
  <ul className="list-disc marker:text-orange-500">
    <li></li>
    ...
  </ul>
</div>
```
See this in action in this [Tailwind play](https://play.tailwindcss.com/kIJoZ5Wbvk)

---

While we're here, some other things to note about your example code. The `prose` class from `@tailwindcss/typography` should be a wrapper around content, **not on it**:

```jsx
<div className="prose">
  <ul>
    <li></li>
    ...
  </ul>
</div>
```

The selector in your CSS is missing a prefix dot, `list-disc` tries to select `<list-disc>` elements but I think you were meant `.list-disc` which matches `<element class="list-disc">`.
